test/data/emb_ruby.conf

```diff
@@ -0,0 +1,14 @@
+<source>
+    @type tail
+    @label @raw
+</source>
+
+<match test>
+    @type forward
+    <secondary>
+ 	@type secondary_file 	
```

Comment:
Check the formatting on these test files.  It looks like there are some tabs that are messing with the indentation. I think just use spaces instead.

Reply:
Good catch, will double check all the files!